src/plugins/intel_gpu/tests/unit/passes/prepare_primitive_fusing_test.cpp

```diff
@@ -688,4 +688,93 @@ TEST(prepare_primitive_fusing, dont_fuse_eltwise_to_dyn_dts) {
 
     ASSERT_NE(prog, nullptr);
     ASSERT_TRUE(has_node(*prog, "scale1"));
-}
\ No newline at end of file
+}
+
+static void move_next(program& p, primitive_id node_id, primitive_id key_id) {
+    auto n = &p.get_node(node_id);
+    p.get_processing_order().erase(&p.get_node(node_id));
+    p.get_processing_order().insert_next(&p.get_node(key_id), n);
+}
+
+TEST(prepare_primitive_fusing, fuse_in_direction_with_lower_processing_number) {
+    //
+    //    in1    in2                  in1    in2                  in1   in2
+    //     |      |                    |      |                    |     |
+    //   conv1  conv2                conv1    |                    |   conv2
+    //    /   \   |        before      |  \   |       after        |  /
+    //  eltw1   eltw4      ----->      |   conv2      ----->     conv1
+    //    |       |                    |   /                       |
+    //  eltw2     |                  eltw5                       eltw6
+    //    |       |                    |
+    //  eltw3 -- eltw5               eltw6
+    //            |
+    //           eltw6
+    //
```

Comment:
this is an inappropriate description to be inserted into the test code. The "center diagram" is a wrong situation and this is not something to be described in the test code. The purpose of test is to validate the left diagram works fine as right diagram.

Reply:
I agree and removed the center diagram.